### PR TITLE
Added WebPush Variant configuration

### PR DESCRIPTION
**Pull request by ziccardi**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-10143

## What
Added documentation for WEBPUSH VARIANT creation

### CODE REVIEW

File: website/docs/configuring_variants.md [hunk 1]
```diff
@@ -4,7 +4,28 @@ title: Configuring Variants
 ---
 
 ## WebPush
-    AEROGEAR-10143
+ The following step-by-step guide gives you an introduction on how to use the _AeroGear UnifiedPush Server_ for sending 
+ _Web Push Notifications_ to your web application.
+ 
+### Create WebPush Variants
+
+ Before you can create a _WebPush Variant_ into the _UnifiedPush Server_ you need to get your VAPID Keys
```
Comment (secondsun) [inline website/docs/configuring_variants.md]: /s/into/with

File: website/docs/configuring_variants.md [hunk 2]
```diff
@@ -4,7 +4,28 @@ title: Configuring Variants
...
+
+ Before you can create a _WebPush Variant_ into the _UnifiedPush Server_ you need to get your VAPID Keys 
+ (see [The Web Push Protocol](https://developers.google.com/web/fundamentals/push-notifications/web-push-protocol) for details).
+ You can use, for example, [this site](https://tools.reactpwa.com/vapid) to generate them.
+
+ As soon as you have the keys, open (or create) an UPS application and click on the `Add A Variant` button
+
+ > ![Add Variant](assets/webpush/add_variant.png "Add Variant")
+
+ A popup will open. Select `WebPush` and fill the data:
```
Comment (secondsun) [inline website/docs/configuring_variants.md]: fill in the data

File: website/docs/configuring_variants.md [hunk 3]
```diff
@@ -4,7 +4,28 @@ title: Configuring Variants
...
+ * Vapid Public Key: your vapid public key
+ * Vapid Private Key: your vapid private key
+ * Alias: The application server contact informations (this must be a `mailto` or an `https` url)
```
Comment (secondsun) [inline website/docs/configuring_variants.md]: information

**Comment by ziccardi** [comment]
@secondsun mind giving another look?

**Pull request merged**